flink/v1.15/flink/src/test/java/org/apache/iceberg/flink/source/enumerator/TestContinuousSplitPlannerImpl.java

```diff
@@ -337,12 +337,10 @@ public void testIncrementalFromSnapshotIdWithEmptyTable() throws Exception {
     ContinuousSplitPlannerImpl splitPlanner =
         new ContinuousSplitPlannerImpl(
             tableResource.tableLoader().clone(), scanContextWithInvalidSnapshotId, null);
-
-    AssertHelpers.assertThrows(
-        "Should detect invalid starting snapshot id",
-        IllegalArgumentException.class,
-        "Start snapshot id not found in history: 1",
-        () -> splitPlanner.planSplits(null));
+    Assertions.assertThatThrownBy(() -> splitPlanner.planSplits(null))
+        .as("Should detect invalid starting snapshot id")
```

Comment:
not needed because it's implicit in the error msg

Reply:
Changes made as suggested

Reply:
unresolving this as it hasn't been addressed